### ISSUE TITLE
Add batch_norm kwarg to DenseNetwork

### ISSUE THREAD

**Issue by brendanhasz** [brendanhasz/probflow#38]
Add `batch_norm` kwarg to DenseNetwork (a bool).

Also additional kwarg, `batch_norm_loc` either `after` (adds batch norm after the activation function, the default) or `before` (adds batch norm before the activation func, as originally suggested).
